.github/dependabot.yml

```diff
@@ -17,6 +17,8 @@ updates:
     - dependency-name: "change-case"
     # see https://github.com/loculus-project/loculus/issues/4020
     - dependency-name: "flowbite-react"
+    - dependency-name: "@types/node"
+      versions: [">=23"] # Increase when we update node version in .nvmrc
```

Comment:
I could see that this will be potentially forgotten - can we add a comment to the .nvmrc file to update this as well if it changes (I assume that will be a manual update)

Reply:
ha, i had the same thought and no we can't add comments to the nvmrc 🤷‍♂️

Reply:
Apparently you can now https://github.com/nvm-sh/nvm/commit/29dce5edfd0976f9a1728c5746715c24061fd404
(obvs would need recent nvm version)

Reply:
Done